### PR TITLE
Update CHANGELOG for v0.1.6

### PR DESCRIPTION
**Pull request by hackerwins**
**What this PR does / why we need it**:

Update CHANGELOG for v0.1.6

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation**:

```docs

```

**Checklist**:
- [x] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

**Review by mojosoeun — APPROVED** [review]
LGTM!

**Review by dc7303 — APPROVED** [review]
LGTM 👍

**Review by ppeeou — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,21 @@ and Yorkie adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
...
 ## [Unreleased]
 
+## [0.1.6] - 2021-07-24
+
+### Added
+- Add Cluster Mode: #183
+- Add Authorization check to Watch API: #209
+- Add authorization-webhook-methods flag: #193
+- Add retry logic to Authorization webhook: #194
+
```
Comment (hackerwins) [inline CHANGELOG.md:17]:
```suggestion
- Add authorization webhook cache: #192

```

**Pull request merged**